pkg/apps/utils/common.go

```diff
@@ -54,3 +58,82 @@ func SetLimits(containerConfig *api.ContainerConfig, config types.Container) {
 		containerConfig.MaxMem = fmt.Sprintf("%dM", *config.MaxMem)
 	}
 }
+
+func GenPVC(b blackduckapi.Blackduck, defaultPVC map[string]string) ([]*components.PersistentVolumeClaim, error) {
```

Comment:
This is a black duck specific function.  I would make it attached to a blackduck related entity under pkg/apps/blackduck.  IMO, creating a base struct with
 	config     *protoform.Config
	kubeClient *kubernetes.Clientset
	blackduck  *blackduckapi.Blackduck

and get GenPVC function, and have that embedded into the v[1|2]/BdPVC struct.